tensorboard/compat/proto/BUILD

```diff
@@ -12,6 +12,23 @@ filegroup(
     srcs = glob(["*.proto"]),
 )
 
+py_test(
+    name = "proto_test",
+    size = "small",
+    srcs = ["proto_test.py"],
+    srcs_version = "PY3",
+    tags = [
+        "manual",
+        "notap",
+    ],
+    deps = [
+        ":protos_all_py_pb2",
+        "//tensorboard:expect_tensorflow_installed",
+    ],
+)
+
+# Rest of file AUTOMATICALLY GENERATED by tensorboard/compat/proto/update.sh.
```

Comment:
Perhaps it might be good to add a line to this comment, saying not to modify the "AUTOMATICALLY GENERATED" substring, which our script explicitly checks for.  e.g. maybe a "do not modify anything below this line manually", and ensure the "AUTOMATICALLY GENERATED" string is below that line.

Reply:
I’m not sure that I understand. Are you saying that the “automatically
generated; do not modify” comment should itself have a comment saying
not to modify the comment, either? IMHO, this seems more confusing than
it’s worth. If it does become a problem, having a `--build-only` mode
and CI check would more robustly ensure that the structure is kept
intact, as mentioned in the PR description.

Reply:
Sure; done.